vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -926,6 +972,113 @@ - (void)onAdRewardsChanged
   // Not needed on iOS because ads do not show unless you are viewing a tab
 }
 
+- (void)setBooleanPref:(const std::string&)path value:(const bool)value
+{
+  const auto key = [NSString stringWithUTF8String:path.c_str()];
+  self.prefs[key] = [NSNumber numberWithBool:value];
+  [self savePrefs];
+}
+
+- (bool)getBooleanPref:(const std::string&)path
+{
+  const auto key = [NSString stringWithUTF8String:path.c_str()];
+  if (![self.prefs objectForKey:key]) {
+    return NO;
+  }
+
```

Comment:
Redundant: `boolValue` returns `NO` if the key doesn't exist

Reply:
Fixed